### PR TITLE
{bio,numlib}[intel/2018a] Beast v1.10.1, beagle-lib v3.0.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2001.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b35a7dd118bd0debb2fca24b29ceff74 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2610.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a41ba07e5ba609aac24bc98d07fa9dd8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2001.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b1411e99c1ffaa58dc77a73080247999 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.11
See https://gist.github.com/59055b2934d6bfd1c1a85948a033a2fe for a full test report.

File: easybuild/easyconfigs/b/Beast/Beast-1.10.1-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+easyblock = 'Tarball'
```
Comment (verdurin) [inline easybuild/easyconfigs/b/Beast/Beast-1.10.1-intel-2018a.eb]: @boegel please re-instate the original credit

Comment (boegel) [reply easybuild/easyconfigs/b/Beast/Beast-1.10.1-intel-2018a.eb]: @verdurin fixed with 43a75d405

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2004.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/89c9522fc660b791f148a97777424d68 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**